src/javascript/JContent/ContentRoute/ToolBar/ToolBar.jsx

```diff
@@ -31,23 +31,25 @@ export const ToolBar = () => {
                 <BrowseControlBar showActions={selection.length === 0}/>}
             {selection.length > 0 &&
             <React.Fragment>
-                <Typography variant="caption" data-cm-role="selection-infos" data-cm-selection-size={selection.length}>
-                    {t('jcontent:label.contentManager.selection.itemsSelected', {count: selection.length})}
-                </Typography>
-                <div className={styles.spacer}/>
-                <Button icon={<Cancel/>} variant="ghost" size="small" onClick={() => dispatch(cmClearSelection())}/>
+                <div className={`flexRow ${styles.selection}`}>
+                    <Typography variant="caption" data-cm-role="selection-infos" data-cm-selection-size={selection.length}>
+                        {t('jcontent:label.contentManager.selection.itemsSelected', {count: selection.length})}
+                    </Typography>
+                    <div className={styles.spacer}/>
+                    <Button icon={<Cancel/>} variant="ghost" size="small" onClick={() => dispatch(cmClearSelection())}/>
```

Comment:
Looks like you forgot to update this button size ('small' while others are using 'default')

Reply:
I was also wondering, but looking at the mockups it seems to be smaller than the others indeed ...